src/protocols/ssh/settings.h

```diff
@@ -54,6 +54,11 @@
 #define GUAC_SSH_DEFAULT_RECORDING_NAME "recording"
 
 /**
+ * The default polling timer for SSH activity in milliseconds.
```

Comment:
I don't get the use of "timer" in referring to the polling timeout. A timer is an object, not a length of time. There must be a better way to word this which matches standard terminology.

My suggestion would be "timeout", but I'm open to anything that makes reasonable sense.